Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -145,30 +150,11 @@ codeunit 4022 "GP Vendor Migrator"
     local procedure MigrateVendorDetails(GPVendor: Record "GP Vendor"; VendorDataMigrationFacade: Codeunit "Vendor Data Migration Facade")
     var
         CompanyInformation: Record "Company Information";
-        GPVendorAddress: Record "GP Vendor Address";
-        HelperFunctions: Codeunit "Helper Functions";
         PaymentTermsFormula: DateFormula;
         VendorName: Text[50];
         ContactName: Text[50];
         Country: Code[10];
     begin
-        // If the Remit To address is found, make that the main address
-        GPVendorAddress.SetRange(VENDORID, GPVendor.VENDORID);
```

Comment:
Did we remove this code intentionally?

Reply:
Yes, part of the Remit To change was to revert back to the original logic for how the primary Vendor address was chosen.